app/assets/javascripts/discourse/app/components/site-header.js

```diff
@@ -279,7 +281,38 @@ const SiteHeaderComponent = MountWidget.extend(
 
       const header = document.querySelector("header.d-header");
       this._itsatrap = new ItsATrap(header);
-      this._itsatrap.bind(["right", "left"], (e) => {
+      this._itsatrap.bind(["right", "left"], (e) =>
+        this._handleArrowKeysNav(e)
+      );
+    },
+
+    _handleArrowKeysNav(event) {
+      if (this.currentUser?.redesigned_user_menu_enabled) {
+        const activeTab = document.querySelector(
```

Comment:
Should we still bind on `right` and `left` for the new menu? It seems like up and down might be more appropriate now that the menu is vertical.

Reply:
Also, I think an acceptance test here to test this behaviour is going to be useful.

Reply:
Changed it to up and down, and I do have a test in the original big PR here https://github.com/discourse/discourse/pull/17379/files#diff-72a374594e8f785149e79288c356e408f9e50eab29b9b5a321ef44da97a8e651R124

I'll add the test in in a later PR because this PR adds the menu with only 1 tab.

Reply:
Ok sounds good :+1: